custom_components/tahoma/coordinator.py

```diff
@@ -146,7 +147,7 @@ async def _async_update_data(self) -> Dict[str, Device]:
                 del self.executions[event.exec_id]
 
         if not self.executions:
```

Comment:
@iMicknl IMO here can be the issue for the too many request. If we missed an event, the executions list will be never empty.

Reply:
Let's have a better look. Maybe we should make sure to reset the update interval after we have to refresh all devices. I thought this logic was somewhere, but I guess this PR never got merged in.